coops-ui/tests/unit/Directors.spec.ts

```diff
@@ -468,6 +475,30 @@ describe('Directors as a BCOMP', () => {
     expect(vm.directors[1].id).toEqual(2)
   })
 
+  it('non-compliance due to too few directors displays accordingly', () => {
+    expect(vm.complianceMsg).toBeNull()
+    expect(vm.$el.querySelector('.complianceCard')).toBeNull()
+    expect(vm.$el.querySelector('.warning-text')).toBeNull()
+    // click first director's cease button
+    click(vm, '#director-1-cease-btn')
+    Vue.nextTick(() => {
+    // click second director's cease button
+      click(vm, '#director-2-cease-btn')
+        Vue.nextTick(() => {
+          expect(vm.complianceMsg.msg).toContain('A minimum of one director is required')
+          expect(vm.$el.querySelector('.complianceCard')).not.toBeNull()
+          expect(vm.$el.querySelector('.warning-text')).not.toBeNull()
+          // un-cease director
+          click(vm, '#director-2-cease-btn')
+          Vue.nextTick(() => {
```

Comment:
When you use "Vue.nextTick()" it's like using setTimeout with a callback function. This callback function (or in your case, arrow function) is called in a future slice of time... after your test has completed!

Your test has completed because it ran code to register a callback to run "in the next tick". Registering this callback succeeded.

The point is, the callback hasn't been run by the time your test completes.

To handle this, the second parameter of the "it()" should be "done" instead of "()". And then, when your test is totally complete (eg, after line 496) then you call `done()`.

There are other example of similar tests. Search for "done()".

In some cases, you can "await" for something to happen (like a click). For examples of that, search for tests with "async" in them.